pypesto/profile/validation_intervals.py

```diff
@@ -50,34 +50,28 @@ def validation_profile_significance(
     problem_full_data:
         pypesto.problem, such that the objective is the
         negative-log-likelihood of the training and validation data set.
-
     result_training_data:
         result object from the fitting of the training data set only.
```

Comment:
```suggestion
        Result object from the fitting of the training data set only.
```
or adjust as other comment